index.html

```diff
@@ -0,0 +1,12 @@
+<!-- index.html -->
```

Comment:
your html should be better after this few weeks ;)

Reply:
I hope you mean mostly the loss of html declaration in this particular example :) I made the changes, but I didn't change anything in the content (body) of this page because thought that this simple page has to give us only no more than a possibility to test our javascript. If you expect something more let me know. I will change it as well.